util/filesort/main_test.go

```diff
@@ -0,0 +1,13 @@
+package filesort
```

Comment:
We need a license header here. See other `main_test.go` file as an example.

Reply:
Thank you for your kind feedback. I will update with the license header.